app/views/devise/mailer/email_changed.html.haml

```diff
@@ -0,0 +1,8 @@
+%p
+  Hello #{@email}!
+- if @resource.try(:unconfirmed_email?)
+  %p
+    We're contacting you to notify you that your email is being changed to #{@resource.unconfirmed_email}.
+- else
+  %p
+    We're contacting you to notify you that your email has been changed to #{@resource.email}.
```

Comment:
```suggestion
%p Hello #{@resource.username}.
- if @resource.try(:unconfirmed_email?)
  %p Your email is being changed to #{@resource.unconfirmed_email}.
- else
  %p Your email has been changed to #{@resource.email}.
```
Possibly some instructions here about what to do if this wasn't you?

Reply:
```suggestion
%p
  Hello #{@email}!
- if @resource.try(:unconfirmed_email?)
  %p
    We're contacting you to notify you that your email is being changed to #{@resource.unconfirmed_email}.
- else
  %p
    We're contacting you to notify you that your email has been changed to #{@resource.email}.
%p If you didn't request this, please look at resetting your password and get in touch with us!
```

Reply:
Okay if we merge those...
```suggestion
%p Hello #{@resource.username}.
- if @resource.try(:unconfirmed_email?)
  %p Your email is being changed to #{@resource.unconfirmed_email}.
- else
  %p Your email has been changed to #{@resource.email}.
%p If you didn't request this, please look at resetting your password and get in touch with us!
```

Reply:
oops, I thought I pulled your suggestion, not the devise one - committing here!